incubator/exporter-otlp/src/test/java/io/aklivity/zilla/runtime/exporter/otlp/internal/OtlpExporterHandlerTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.exporter.otlp.internal;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.mock;
+
+import java.util.List;
+import java.util.function.LongFunction;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import io.aklivity.zilla.runtime.engine.EngineConfiguration;
+import io.aklivity.zilla.runtime.engine.EngineContext;
+import io.aklivity.zilla.runtime.engine.config.AttributeConfig;
+import io.aklivity.zilla.runtime.engine.config.ExporterConfig;
+import io.aklivity.zilla.runtime.engine.config.KindConfig;
+import io.aklivity.zilla.runtime.engine.config.OptionsConfig;
+import io.aklivity.zilla.runtime.engine.metrics.Collector;
+import io.aklivity.zilla.runtime.exporter.otlp.internal.config.OtlpEndpointConfig;
+import io.aklivity.zilla.runtime.exporter.otlp.internal.config.OtlpExporterConfig;
+import io.aklivity.zilla.runtime.exporter.otlp.internal.config.OtlpOptionsConfig;
+import io.aklivity.zilla.runtime.exporter.otlp.internal.config.OtlpOverridesConfig;
+import io.aklivity.zilla.runtime.exporter.otlp.internal.config.OtlpSignalsConfig;
+
+public class OtlpExporterHandlerTest
```

Comment:
Suggest this test logic will migrate (and simplify) to `OtlpEndpointConfigTest` instead after `URI` and `resolveXXX` method changes to `OtlpEndpointConfig`.

Reply:
yep, done